core/psh/ping/ping.c

```diff
@@ -103,36 +103,35 @@ static int psh_ping_socketConfig(void)
 }
 
 
-static void psh_ping_reqInit(uint8_t *data, int len)
+static void ping_reqinit(uint8_t *data, int len)
 {
-	struct icmphdr *hdr = (struct icmphdr *) data;
+	struct icmphdr *hdr = (struct icmphdr *)data;
 	int i;
 
 	hdr->type = ICMP_ECHO;
 	hdr->un.echo.id = htons(getpid());
 
 	for (i = 0; i < len - sizeof(struct icmphdr); i++)
-		data[sizeof(struct icmphdr) + i] = (uint8_t) i;
+		data[sizeof(struct icmphdr) + i] = (uint8_t)i;
 }
 
 
-static int psh_ping_send(int fd, uint8_t *data, int len)
+static int ping_send(int fd, uint8_t *data, int len)
 {
-	struct icmphdr *hdr = (struct icmphdr *) data;
+	struct icmphdr *hdr = (struct icmphdr *)data;
 
 	hdr->un.echo.sequence = htons(ping_common.seq++);
 	hdr->checksum = 0;
-	hdr->checksum = htons(psh_ping_chksum(data, len));
+	hdr->checksum = htons(ping_chksum(data, len));
 
-	if (sendto(fd, data, len, 0, (struct sockaddr *) &ping_common.raddr,
-	           sizeof(ping_common.raddr)) != len)
+	if (sendto(fd, data, len, 0, (struct sockaddr *)&ping_common.raddr, sizeof(ping_common.raddr)) != len)
```

Comment:
You can break lines no problem, just use two indetations for broken line (i.e. two tabs)